components/earnings/commons/earnings_base.lua

```diff
@@ -0,0 +1,187 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Earnings/Base
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Earnings = {}
+local MathUtils = require('Module:Math')
+local String = require('Module:StringUtils')
+local Logic = require('Module:Logic')
+local Class = require('Module:Class')
+
+local _DEFAULT_DATE = '1970-01-01 00:00:00'
+local _MAX_QUERY_LIMIT = 5000
+
+-- customizable in /Custom
+Earnings.defaultNumberOfPlayersInTeam = 5
+
+---
+-- Entry point for players and individuals
+-- @player - the player/individual for whom the earnings shall be calculated
+-- @year - (optional) the year to calculate earnings for
+-- @mode - (optional) the mode to calculate earnings for
+-- @noRedirect - (optional) player redirects get not resolved before query
+-- @prefix - (optional) the prefix under which the players are stored in the placements
+-- @playerNumber - (optional) the number for how many params the query should look in LPDB
```

Comment:
Name of this field could be improved

Reply:
give me a suggestion^^

Reply:
`playerPositionLimit`

Reply:
`maxPlayers` or `maxNumPlayers` are the first ones that I think of. Not the best names neither, but better then `playerNumber`

Reply:
went with vogans suggestion